src/js/components/Menu.js

```diff
@@ -39,6 +39,25 @@ var MenuDrop = React.createClass({
       down: this._onDownKeyPress
     };
     this.startListeningToKeyboard(this._keyboardHandlers);
+    var menuItems = this.refs.navContainer.getDOMNode().childNodes;
+    for (var i = 0; i < menuItems.length; i++) {
+      var classes = menuItems[i].className.split(/\s+/);
+      var tagName = menuItems[i].tagName.toLowerCase();
+      if (tagName !== 'button' && tagName !== 'a' && classes.indexOf('check-box') === -1) {
+        continue;
+      }
+      menuItems[i].setAttribute('role', 'menuitem');
+      
+      if (!menuItems[i].getAttribute('id')) {
+        menuItems[i].setAttribute('id', menuItems[i].getAttribute('data-reactid'));
+      }
+      // aria-selected informs AT which menu item is selected for that menu container.
+      if (classes.indexOf('active') > -1) {
+        menuItems[i].setAttribute('aria-selected', true);
+      } else {
+        menuItems[i].setAttribute('aria-selected', false);
```

Comment:
can you add this as: `menuItems[i].setAttribute('aria-selected', classes.indexOf('active') > -1);`